Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -682,7 +682,7 @@ private Result ContextMenuPluginInfo(string id)
             var plugin = translator.GetTranslation("plugin");
             var title = $"{plugin}: {metadata.Name}";
             var icon = metadata.IcoPath;
-            var subtitle = $"{author}: {metadata.Author}, {website}: {metadata.Website} {version}: {metadata.Version}";
+            var subtitle = "";
```

Comment:
i thought we are keeping Author, no?

Reply:
It wasn't decided for sure. I organized it "by author"